src/styles/index.scss

```diff
@@ -1,3 +1,65 @@
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6
+);
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+
 }
+
+.calendar {
+  width: 707px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to avoid setting fixed widths. Consider using max-width for better responsiveness.